cirq/protocols/channel.py

```diff
@@ -107,6 +107,13 @@ def channel(val: Any,
     if channel_result is not NotImplemented:
         return tuple(channel_result)
 
+    mixture_getter = getattr(val, '_mixture_', None)
+    print(mixture_getter)
```

Comment:
Aha! You have to debug your code! I knew it!

Reply:
ha